### ISSUE TITLE
Swift 3.0 Support available on specific branch

### ISSUE THREAD

**Issue by uraimo** [uraimo/Bitter#3]
Tagged as 1.2.0, i'm not going to push the pod yet (something else will surely change before the Swift 3.0 is released).

**Comment by umverteiler** [comment]
Great - but maybe it would be a good idea to change the Version Tag to 2.0.0. We´re using Xcode 7, Swift 2.2 and Carthage. In our Cartfile we set the Bitter Framework to `~= 1.1.2` (i.e. compatible with version 1.1.2). But since you´re 1.2.0 expects Swift 3 / Xcode 8 the build command fails now. Using the Semantic Versioning Carthage would know that 2.0.0 isn´t usable in this case.

We changed the Cartfile to `== 1.1.2` and now it works (again).
